diff --git a/node_modules/expo-modules-core/ios/AppDelegates/ExpoAppInstance.swift b/node_modules/expo-modules-core/ios/AppDelegates/ExpoAppInstance.swift
index 1e6e7ec..67712a3 100644
--- a/node_modules/expo-modules-core/ios/AppDelegates/ExpoAppInstance.swift
+++ b/node_modules/expo-modules-core/ios/AppDelegates/ExpoAppInstance.swift
@@ -43,7 +43,7 @@ open class ExpoAppInstance: RCTAppDelegate {
 
     let configuration = RCTRootViewFactoryConfiguration(
       bundleURLBlock: bundleUrlBlock,
-      newArchEnabled: newArchEnabled(),
+      newArchEnabled: false,
       turboModuleEnabled: turboModuleEnabled(),
       bridgelessEnabled: bridgelessEnabled()
     )
@@ -62,25 +62,7 @@ open class ExpoAppInstance: RCTAppDelegate {
     }
 
     // NOTE(kudo): `sourceURLForBridge` is not referenced intentionally because it does not support New Architecture.
-    configuration.sourceURLForBridge = nil
-
-    if responds(to: #selector(extraModules(for:))) {
-      configuration.extraModulesForBridge = { bridge in
-        return appDelegate.extraModules(for: bridge)
-      }
-    }
-
-    if responds(to: #selector(extraLazyModuleClasses(for:))) {
-      configuration.extraLazyModuleClassesForBridge = { bridge in
-        return appDelegate.extraLazyModuleClasses(for: bridge)
-      }
-    }
-
-    if responds(to: #selector(bridge(_:didNotFindModule:))) {
-      configuration.bridgeDidNotFindModule = { bridge, moduleName in
-        return appDelegate.bridge(bridge, didNotFindModule: moduleName)
-      }
-    }
+    //sda
 
     return ExpoReactRootViewFactory(
       reactDelegate: reactDelegate,
